packages/monaco/src/browser/monaco-bulk-edit-service.ts

```diff
@@ -23,8 +23,29 @@ export class MonacoBulkEditService implements monaco.editor.IBulkEditService {
     @inject(MonacoWorkspace)
     protected readonly workspace: MonacoWorkspace;
 
+    private _previewHandler?: monaco.editor.IBulkEditPreviewHandler;
```

Comment:
do you know what is for and how to test?

Reply:
please see https://github.com/eclipse-theia/theia/pull/8010#issuecomment-673453375